docarray/array/storage/redis/backend.py

```diff
@@ -146,6 +148,13 @@ def _build_schema_from_redis_config(self):
             index_param['INITIAL_CAP'] = self._config.initial_cap
         schema = [VectorField('embedding', self._config.method, index_param)]
 
+        if self._config.index_text:
+            schema.append(TextField('text'))
+
+        if self._config.tag_indices:
+            for index in self._config.tag_indices:
+                schema.append(TextField(index))
```

Comment:
if you check the existence of `tag_indices`, the `tag_indices` you defined in line line 28 should be `Optional[List[str]]`, right? otherwise if the default value is `[]`, checking is not necessary